kinbot/optimize.py

```diff
@@ -161,8 +164,12 @@ def do_optimization(self):
                     # do the high level calculations
                     if self.par['high_level'] == 1:
                         if self.shigh == -1:
+                            if self.species.wellorts:
+                                name = self.species.name
+                            else:
+                                name = self.species.chemid
```

Comment:
same here. Also could you make a small function within optimize that's called get_name(self, species) and returns the name? these lines are repeating.